devicetypes/smartthings/zwave-lock.src/zwave-lock.groovy

```diff
@@ -55,6 +55,12 @@ metadata {
 		fingerprint mfr:"0129", prod:"0001", model:"0409", deviceJoinName: "Yale Touchscreen Lever Door Lock" // YRL-220-ZW-605
 		// Samsung
 		fingerprint mfr:"022E", prod:"0001", model:"0001", deviceJoinName: "Samsung Digital Lock" // SHP-DS705, SHP-DHP728, SHP-DHP525
+		//KeyWe
+		fingerprint mfr:"037B", prod:"0002", model:"0001", deviceJoinName: "KeyWe Lock"
+        //KWIKSET OBSIDIAN
+        //zw:Fs type:4003 mfr:0090 prod:0003 model:0742 ver:4.10 zwv:4.34 lib:03 cc:5E,72,5A,98,73,7A sec:86,80,62,63,85,59,71,70,4E,8B,4C,5D role:07 ff:8300 ui:8300
+        fingerprint mfr:"0090", prod:"0003", model:"0742", deviceJoinName: "OBSIDIAN Lock"
```

Comment:
Indentation is wrong here bc it's using spaces, not tabs

Reply:
Also the join name should probably be `Kwikset Obsidian Lock`